docs/source/release_notes/release-notes.rst

```diff
@@ -81,46 +117,6 @@ develop
            This change was made to simplify the API and also remove any illusion that specifying the ``partitionStrategy`` would have done anything.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2864>`__)
 
-    *    - |devbreak|
-         - Removed ``CassandraKeyValueServiceConfigManager``. If you're affected by this, please contact the AtlasDB team.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2872>`__)
-
-    *    - |improved| |metrics|
-         - BackgroundSweeperImpl now logs if there's an uncaught exception.  Added 2 new outcomes for normal and abnormal shutdown to allow closer monitoring.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2884>`__)
-
-    *    - |fixed|
-         - Qos clients will query the service every 2 seconds instead of every client request. This should prevent too many requests to the service.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2872>`__)
-
-    *    - |fixed| |userbreak|
-         - AtlasDB will now *fail to start* if a TimeLock block is included in the initial runtime configuration, but the install configuration is set up with a leader block or with remote timestamp and lock blocks.
-           Previously, AtlasDB would start successfully under these conditions, but the TimeLock block in the runtime configuration would be silently ignored.
-           Note that the decision on whether to use TimeLock or another source of timestamps and locks is made at install-time.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2850>`__)
-
-    *    - |improved| |userbreak|
-         - AtlasDB users can now specify the usage of TimeLock Server purely by including a TimeLock block in the initial runtime configuration.
-           Previously, AtlasDB users would need to specify that they were using TimeLock in the install configuration, possibly with an empty object (``timelock: {}``).
-           This is a change from previous behaviour in cases where users specified an embedded install configuration but a TimeLock block in the runtime configuration; previously, the embedded configuration would have been selected, while now the TimeLock configuration will be selected.
-
-           Also, users with scripts that depend on supplying a default runtime configuration may need to be careful to ensure that TimeLock configuration is preserved when such scripts are run.
-           That said, AtlasDB will fail to start if trying to access a key-value service where TimeLock has been used as a source of timestamps without going through TimeLock, so we don't think there is a risk of data corruption.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2850>`__)
-
-    *    - |fixed|
```

Comment:
Did this get remerged?  I don't see it in the release notes?

Reply:
If you're referring to 2874, then it's down there, third entry on 0.73.1.

Reply:
I'm just blind, ignore me and merge!